feat(#2515): add option to change grouped folders name with custom function (#2521)
nvim-tree/nvim-tree.lua: doc/nvim-tree-lua.txt

*nvim-tree.renderer.group_empty*
Compact folders that only contain a single folder into one node.

> Can be a custom function that takes one argument (the relative path

Reviewer: ```suggestion
Boolean or a function that takes single argument (the relative path
```

Author: I agree with all the corrections, I would just leave "one" instead of "single" since it's more consistent with two other occurrences of something like "the function takes _one_ argument" throughout the docs

Reviewer: Absolutely, consistency is the key. Thanks!

> of the grouped folders) and returns a string (the newly elaborated name

Reviewer: ```suggestion
of grouped folders) and returns a string to be displayed.
```

Not 100% sure about grammar here.

> that will be shown in the tree)

Reviewer: ```suggestion

```
